vol_size: 0.24  # 4 times the max gripper opening width
threshold: 0.9
